xDrip/Scenes/Entries/EditCalibration/EditCalibrationPresenter.swift

```diff
@@ -22,7 +22,112 @@ final class EditCalibrationPresenter: EditCalibrationPresentationLogic {
     // MARK: Do something
     
     func presentLoad(response: EditCalibration.Load.Response) {
-        let viewModel = EditCalibration.Load.ViewModel(displaySecondEntrySet: !response.hasInitialCalibrations)
+        var sections: [BaseSettings.Section] = [
+            createSection(.firstInput, response: response)
+        ]
+        
+        if !response.hasInitialCalibrations {
+            sections.append(
+                createSection(.secondInput, response: response)
+            )
+        }
+        
+        let tableViewModel = BaseSettings.ViewModel(sections: sections)
+        let viewModel = EditCalibration.Load.ViewModel(tableViewModel: tableViewModel)
         viewController?.displayLoad(viewModel: viewModel)
     }
+    
+    private func createSection(
+        _ field: EditCalibration.Field,
+        response: EditCalibration.Load.Response
+    ) -> BaseSettings.Section {
+        let cells: [BaseSettings.Cell] = [
+            createGlucosePickerCell(field, glucoseValueChangedPicker: response.glucosePickerValueChanged),
+            createDatePickerCell(field, dateValueChagedHandler: response.datePickerValueChanged)
+        ]
+        
+        var header = ""
+        var footer: String?
```

Comment:
It doesn't do a big difference in such scale, but to avoid double value setting, you could declare these vars as `let`. E.g. currently header has the initial value of `""`, which is not used and is overridden in any case.
Replace with
```swift
let header: String
let footer: String?
```